drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -1791,7 +1823,32 @@ static int vidioc_s_selection(struct file *file, void *priv,
 		}
 		break;
 	case ISP:
+		break;
 	case DEINTERLACE:
+		switch (s->target) {
```

Comment:
These need some extra validation. COMPOSE targets are only valid on V4L2_BUF_TYPE_VIDEO_CAPTURE, and CROP targets are only valid on V4L2_BUF_TYPE_VIDEO_OUTPUT

Reply:
Whilst fixed for g_selection, this still applies for s_selection